### ISSUE TITLE
Autoplay:false not working

### ISSUE THREAD

**Issue by devalphabeta** [carloscabo/responsive-youtube-player-with-playlist#10]
I am loading a bunch of single video IDS and autoplay is always active.  How do I disable the autoplay, I have tried autoplay:false but this has no effect.

```
<div id="rypp-1388292409" class="RYPP r16-9" data-ids="6ZJXaBDZhcc">
<div class="RYPP-video">
    <div class="RYPP-video-player">
    <!-- Will be replaced -->
    </div>
</div>
<div class="RYPP-playlist">
    <header>
        <h2 class="_h1 RYPP-title">Title</h2>
        <p class="RYPP-desc">Description</p>
    </header>
    <div class="RYPP-items"></div>
</div>
</div>

</div>

```

**Comment by sebaghar** [comment]
Same Problems for me..I also keep the auto play false not not working. here is the code // Api key
  this.api_key = api_key;

  // Default options
  this.options = {
    update_title_desc: false,
    autoplay: false,
    autonext: false,
    loop: false,
    mute: false,
     // Logs debug messages in console
    debug: false     // Default
  };

**Comment by sebaghar** [comment]
Hi ! You can see my code and then use it. It is solved.Please use this code for solving the issues.Just copy this and paste at your RYPP.js Code :http://news365.bdtask.com/Demo/DemoNews365/js/RYPP.js

**Comment by carloscabo** [comment]
Fixed in V2.22
